index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+        <main id="main-doc">
+                  <h1>Agile and waterfall approaches</h1d>
```

Comment:
- [ ] Please i suggest you reduce the indentations and extra spacings before your tags, for clearer code readability